test/common_test/joint_features.cc

```diff
@@ -1146,22 +1146,201 @@ TYPED_TEST(JointFeaturesAttachDetachTest, JointAttachDetach)
     }
 
     // After a while, body2 should reach the ground and come to a stop
-    std::size_t stepCount = 0u;
-    const std::size_t maxNumSteps = 2000u;
-    while (stepCount++ < maxNumSteps)
+    for (std::size_t i = 0; i < 1000; ++i)
     {
       world->Step(output, state, input);
+    }
+    frameDataModel2Body = model2Body->FrameDataRelativeToWorld();
+    EXPECT_NEAR(0.0, frameDataModel2Body.linearVelocity.z(), 1e-3);
+  }
+}
+
+/////////////////////////////////////////////////
+// Attach a fixed joint between links from non-static models to a link
+// from a model fixed to the world. Verify the models are not moving when
+// the fixed joint is attached.
+TYPED_TEST(JointFeaturesAttachDetachTest, JointAttachDetachFixedToWorld)
+{
+  for (const std::string &name : this->pluginNames)
+  {
+    std::cout << "Testing plugin: " << name << std::endl;
```

Comment:
remove ? or include `<iostream>`

Reply:
kept for consistency with other tests. Added include in [2e49b03](https://github.com/gazebosim/gz-physics/pull/632/commits/2e49b03d4203df43d6ba10e8577a8c0cfe17e04b)